Source: Andreas Sturmlechner, http://sourceforge.net/tracker/?func=detail&aid=2752871&group_id=6488&atid=306488
Upstream: Ticket isn't resolved yet, I couldn't find a VCS for krusader
Reason: Build with GCC 4.4.0

--- a/krusader/VFS/kiojobwrapper.cpp    2009-04-11 12:52:31.318455396 +0200
+++ b/krusader/VFS/kiojobwrapper.cpp 2009-04-11 12:55:12.005493271 +0200
@@ -28,6 +28,7 @@
  *                                                                         *
  ***************************************************************************/

+#include <stdio.h>
 #include "kiojobwrapper.h"
 #include <qevent.h>
 #include <kurl.h>
